src/services/formatting/formatting.ts

```diff
@@ -341,6 +341,14 @@ namespace ts.formatting {
             processNode(enclosingNode, enclosingNode, startLine, undecoratedStartLine, initialIndentation, delta);
         }
 
+        if (!formattingScanner.isOnToken()) {
+            let leadingTrivia = formattingScanner.readTokenInfo(undefined).leadingTrivia;

```

Comment:
this is not a supported way of using readTokenInfo, i guess you need a new API on the formattingScanner to getCurrrentLeadingTriva instead.